community.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Community"
+---
+
+## An Open Repository for R Packages
+
+Packages are available for installation from the repository at <https://community.r-multiverse.org>.
+
+A traditional software version cycle begins with the development phase, where bugs are identified and fixed, and ends with the production phase, which promises a seamless user experience.
+
+Community aims at a space just before production, commonly referred to as Quality Assurance (QA). This means that:
+
+1. Each package release has the full endorsement of its own maintainer. R-multiverse always provides a version that its developer chose to distribute for general use.
+2. The user is still responsible for judging whether a package is appropriate to use. Some releases may still have issues, as indicated by its check results, publicly available at <https://community.r-multiverse.org>.
+
+## Continuous maintainer-driven deployment
+
+Anyone can [contribute](contributors.qmd) packages to Community via GitHub. Each package only needs to be registered once.
```

Comment:
line breaks after sentences, I promise it makes future diffs worth it :smile_cat:

Reply:
Thanks! I will definitely get into this habit. Have made changes elsewhere as well to implement this.